libraries/engage/cart/components/CartItems/CartItemGroupReservation.jsx

```diff
@@ -1,29 +1,31 @@
+// @flow
 import React from 'react';
-import PropTypes from 'prop-types';
 import { every, isEmpty } from 'lodash';
 import CardListItem from '@shopgate/pwa-ui-shared/CardList/components/Item';
 import { Accordion } from '@shopgate/engage/components';
 import { StoreAddressOpeningHours, StoreAddressPhoneNumber } from '@shopgate/engage/locations';
+import type { OptionalLocationAware } from '@shopgate/engage/locations/locations.types';
```

Comment:
Please change to `import { type OptionalLocationAware } from '@shopgate/engage/locations/locations.types'`. The `type` has to be inside the `{}` before every imported element.

Reply:
It was intentional, since *.types content only types and is empty. But ok